aide-de-camp-sqlite/README.md

```diff
@@ -82,7 +82,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let queue = SqliteQueue::with_pool(pool);
 
     // Add job the queue to run next
-    let _jid = queue.schedule::<MyJob>(vec![1,2,3]).await?;
+    let _jid = queue.schedule::<MyJob>(vec![1,2,3], 0).await?;
```

Comment:
This is unrelated to the changes in this PR, but the doc tests were failing because of this missing parameter.